qlf-k6n10f: DSP ports fix

lpawelcz opened the pull request:
This PR is a follow up to https://github.com/chipsalliance/yosys-f4pga-plugins/pull/318. It rearranges `feedback`, `output_select` and `register_inputs` ports in DSP simulation models, adds `acc_fir` to `QL_DSP2_MULTADD*` cells and removes unnecessary `clk` from `QL_DSP2_MULTADD`. @rakeshm75, @tpagarani - please have a look at this and let me know if everything is ok.

rakeshm75 commented:
@lpawelcz , Why is clk port removed in QL_DSP2_MULTADD module in cells_sim.v file?

tpagarani commented:
Can the Conversion of DSP static inputs to Mode bits be separated from the other fixes that have been done to pass down required inputs and removal of redundant ports?

Please create a separate one for Conversion to Mode bits.

lpawelcz commented:
@tpagarani sorry, I forgot to mention that here: I rebased this PR on top of https://github.com/chipsalliance/yosys-f4pga-plugins/pull/360 which actually contains only the conversion to `MODE_BITS` (+2 new test cases).
That is why all commits from those two PRs are visible in diff in this PR. I did that because I believe this PR should be merged right after https://github.com/chipsalliance/yosys-f4pga-plugins/pull/360 so I rebased it to solve conflicts.

lpawelcz commented:
Superseded with https://github.com/chipsalliance/yosys-f4pga-plugins/pull/363
